docs/marketplace/dev/glue-api-guides/202204.0/marketplace-shopping-lists/managing-marketplace-shopping-lists.md

```diff
@@ -363,14 +363,16 @@ To retrieve concrete products in a shopping list, include `shopping-list-items`
 
```

Comment:
shouldn't we add here a separate message that in order to retrieve merchants for the shopping list items, one should inclulde `shopping list items` and `merchants` resrouces? And to retrieve product offers and their availability in a shopping list, one should include `product-offers` and `product-offer-availabilities`?